HLTriggerOffline/Btag/test/config.ini

```diff
@@ -8,42 +8,42 @@ files= root://cms-xrd-global.cern.ch//store/relval/CMSSW_8_1_0_pre7/RelValTTbar_
 
 ;Paths names and btag variables that you want to monitor
 [btag]
-HLT_PFMET120_:
+HLT_PFMET120_PFMHT120_IDTight_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_HT300PT30_QuadJet_75_60_45_40_v:
+HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_PFHT380_SixJet32_DoubleBTagCSV_:
+HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
 HLT_Ele28_eta2p1_WPTight_Gsf_HT150_:
```

Comment:
are you sure about this ?
shouldn't it be
`  HLT_Ele28_eta2p1_WPTight_Gsf_HT150_v`

Reply:
Yes, there is only one HLT path that could be found with this pattern. To be more strict it is of course possible to add _v